docs/source/getting-started/installing.adoc

```diff
@@ -76,6 +76,11 @@ On the Windows operating system, due to issue link:https://github.com/minishift/
 You cannot run {project} from a network drive.
 ====
 
+[NOTE]
+====
+Minishift will use any `ssh` binary found on the `PATH` in preference to internal ssh code. Ensure that any `ssh` binary in the system `PATH` does not generate warning messages as this will cause installation problems such as issue  link:https://github.com/minishift/minishift/issues/1923[#1923].
```

Comment:
One sentence per line, please -- this makes editing significantly easier and keeps diffs clean. :)

Please use the `{project}` attribute instead of the literal name "Minishift". We should capitalize "SSH" in "internal ssh code".